frog-utils/src/generateReactiveFn.js

```diff
@@ -36,6 +39,20 @@ class Doc {
     this.updateFn = updateFn;
   }
 
+  createLearningItem(liType, item, meta) {
+    const id = uuid();
+    const itempointer = this.doc.connection.get('li', id);
+    itempointer.create({
+      liType,
+      payload: item,
+      createdAt: new Date(),
+      ...meta,
+      ...this.meta
```

Comment:
Wouldn't it be better to have ?
```
{
      liType,
      payload: item,
      createdAt: new Date(),
      meta: { 
          ...meta,
          ...this.meta
      }
}
```

So that ac-pkg can safely write things in meta without overwriting other fields?

Reply:
Also as ac-pkg will have access to the dataFn, I thought at first that they were only able to modify the `meta` field. With the full dataFn, we can imagine that an ac-pkg can implement a custom EditorComponent for a LI or even change the `liType`?